sprinkle/high-level/pytorch_trial.py

```diff
@@ -0,0 +1,47 @@
+"""
+High-level Sprinkle API for training with PyTorchTrial.
+"""
+
+# ExperimentConfig settings:
+#
+#   Backwards compatibility, where you only submit a Trial.
+#       #launch_layer: python3 -m determined.launch.auto_horovod
+#       #entrypoint_script: python3 -m determined.exec.harness
+#       entrypoint: model_def:MyTrial
+#
+#   Basic usage:
+#       #launch_layer: python3 -m determined.launch.auto_horovod
+#       entrypoint_script: python3 train.py
```

Comment:
1. It won't be in a startup hook.  It would be an executable of some sort that would launch all of the workers on the node.  We'd call the launch layer and pass it the `entrypoint_script` as args.

2. yes, the context object only exists inside of the harness, but the experiment config is required to launch the experiment in the first place.